### DISCUSSION TITLE
Regarding Row_affected column in model_execution table

### DISCUSSION THREAD

**Discussion by bhratkapoor** [brooklyn-data/dbt_artifacts#207] [Q&A] [answered]
I am using the brooklyn-data package for logging purpose, all setup seems fine but the table MODEL_EXECUTIONS having column ROWS_AFFECTED is not getting any value..even if we execute the any combination of model. Any help will be appreciated.

**Comment by jaypeedevlin** [comment, marked as answer]
Duplicate of https://github.com/brooklyn-data/dbt_artifacts/issues/196
